content-repo/create-docs.sh

```diff
@@ -29,7 +29,7 @@ else
     echo "==== current branch: ${CURRENT_BRANCH} ===="
 
     CONTENT_GIT_URL="https://github.com/demisto/content.git"
-    CONTENT_BRANCH="master"
+    CONTENT_BRANCH="fix-common-server-python-docstring"
```

Comment:
Once [this](https://github.com/demisto/content/pull/35197) PR has been merged, I will change it to master